pytradfri/api/aiocoap_api.py

```diff
@@ -0,0 +1,162 @@
+"""Coap implementation using aiocoap."""
+import asyncio
+import json
+import logging
+import sys
+
+sys.path.insert(0, '/usr/src/build/tinydtls/cython')  # noqa
```

Comment:
This is not something that we should solve inside the lib. Instead we should provide instructions for people to make sure that we have access to the right dependencies.